CONFIG.md

```diff
@@ -46,7 +46,7 @@ For Kubernetes, it is important to set resource limits.
 TODO: explain how to set limits, with default, project and spider specificity.
 
 ### [joblogs] section
-* `logs_dir`     - a directory to store log files collected on k8s cluster (implemented only for Kubernetes). When configuring, keep in mind that in the Dockerfile the `USER` is set to `nobody` so not all directories are writable, but if you make a child directory under `/tmp` you won't encounter permission problems.
+* `logs_dir`     - a directory to store log files collected on k8s cluster (implemented only for Kubernetes). If you are using a Persistent Volume, keep in mind, that the provided path should be mounted in the deployment manifest.
```

Comment:
Great. Maybe also mention that it needs to have read and write permissions, e.g. for Kubernetes by setting the `securityContext`'s `fsGroup` (with a link to the k8s docs, if applicable).